### PR TITLE
Centralize `isConnection`, `isLabel` and `isRoot` Checks

### PR DESCRIPTION
**Pull request by philippfromme**
* export `isConnection`, `isLabel` and `isRoot` checks from `ModelUtil` and test
* use these checks
* checks will be usable by bpmn-js, too

---

Required by https://github.com/bpmn-io/bpmn-js/pull/1903
Related to https://github.com/bpmn-io/bpmn-js/issues/1901

### CODE REVIEW

**Comment by philippfromme** [comment]
@nikku When fixing https://github.com/bpmn-io/bpmn-js/issues/1901 I tried this approach. I'd vote for using actual `instanceof` checks whenever possible and using `is*Like` checks when not (eg. `CopyPaste`). Let me know what you think.

**Comment by philippfromme** [comment]
> 'labelTarget' in someObject

My first idea was using `Object.hasOwn(label, 'labelTarget')`. But this still doesn't prevent bugs like https://github.com/bpmn-io/bpmn-js/issues/1902. The [respective test](https://github.com/bpmn-io/bpmn-js/blob/develop/test/spec/features/rules/BpmnRulesSpec.js#L81) doesn't work because the connection is created without `waypoints`. Checking for `instanceof` would prevent that.

**Comment by philippfromme** [comment]
As discussed with @nikku I will use `Object.hasOwn` instead of `instanceof` as it can lead to issues when dealing with more than one instance of diagram-js.

**Comment by nikku** [comment]
@philippfromme [ `min-dash` `has` you covered ](https://github.com/bpmn-io/min-dash/blob/master/lib/lang.d.ts#L29).

File: test/spec/model/ModelSpec.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
 import {
   create,
-  isModelElement
+  isElementInstance
```
Comment (nikku) [inline test/spec/model/ModelSpec.js:3]: Why would we rename this?

Comment (philippfromme) [reply test/spec/model/ModelSpec.js:3]: We don't have to. One thing I was wondering about is the fact we use this check in `Modeling`. Doesn't that have the same issue with having more than one diagram-js?

Comment (nikku) [reply test/spec/model/ModelSpec.js:3]: Yes, it does. However it was never a problem to date. A safe way is to rather do the check in the `elementFactory`, hence my prior attempt to move it there.

Comment (philippfromme) [reply test/spec/model/ModelSpec.js]: Reverted.

**Pull request merged**